src/openMVG/multiview/solver_resection_p2pt_fabbri.hpp

```diff
@@ -0,0 +1,52 @@
+// This file is part of OpenMVG, an Open Multiple View Geometry C++ library.
+
+// Copyright (c) 2020 Ricardo Fabbri, Ariel Kovaljski
+//
+// Author: Ariel Kovaljski and Ricardo Fabbri
+// Rio de Janeiro State University
+
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#ifndef OPENMVG_MULTIVIEW_RESECTION_P2Pt_FABBRI_HPP
+#define OPENMVG_MULTIVIEW_RESECTION_P2Pt_FABBRI_HPP
+
+#include "openMVG/multiview/two_view_kernel.hpp"
+#include "openMVG/multiview/solver_resection_metrics.hpp"
+
+#if defined(_MSC_VER)
```

Comment:
Should move to CPP since it is not used in the hpp file

Reply:
done.